runtime/lua/vim/lsp/linked_editing_range.lua

```diff
@@ -0,0 +1,199 @@
+local util = require('vim.lsp.util')
+local log = require('vim.lsp.log')
+local ms = require('vim.lsp.protocol').Methods
+local Range = require('vim.treesitter._range')
+local api = vim.api
+local M = {}
+
+---@class (private) vim.lsp.linked_editing_range.globalstate Global state for linked editing ranges
+---@field enabled boolean Whether linked editing ranges are enabled
+---@field word_pattern? string
+---@field range_index? integer
+---@type vim.lsp.linked_editing_range.globalstate
+M.globalstate = {
+  enabled = false,
+}
+
+local globalstate = M.globalstate
+
+local augroup = api.nvim_create_augroup('nvim.lsp.linked_editing_range', {})
+local ns = api.nvim_create_namespace('nvim.lsp.linked_editing_range')
+
+--- |lsp-handler| for the method `textDocument/linkedEditingRange`
+--- Stores ranges globally, since they only make sense to track for the current buffer
+---@param result lsp.LinkedEditingRanges
+---@param ctx lsp.HandlerContext
+---@private
+function M.on_linkededitingrange(err, result, ctx)
```

Comment:
```suggestion
function M.on_linked_editing_range(err, result, ctx)
```

Reply:
Also we should be able to make this a local function if we remove the handler in `handlers.lua`